src/scripts/contraception/scenarios/run_analysis_co_health_impacts_no_diseases.py

```diff
@@ -0,0 +1,124 @@
+"""
+This file defines a batch run to get sims results to be used by an analysis script
+( # TODO: create one and add its name here
```

Comment:
Todo.....?

Reply:
we didn't start working on the analysis script yet, and now this project was postponed